cnd/MIGRATIONS.md

```diff
@@ -0,0 +1,34 @@
+# Migrations in cnd
+
+## Context
+
+This README offers guidelines on how to manage database migrations in cnd.
+
+This document is motivated by the fact, that handling unnecessary migrations adds unnecessary complexity.
+Migrations are only relevant once a new version of cnd is released, because the prior motivation for migrations is ensuring downwards compatibility and tracking changes to achieve this compatibility.
```

Comment:
nit: I think you want 'backwards compatibility' :)

Reply:
Yes, thanks!